spacewalk/config/etc/httpd/conf.d/zz-spacewalk-www.conf

```diff
@@ -6,7 +6,7 @@ Logformat "%t %h %{SSL_PROTOCOL}x %{SSL_CIPHER}x \
 DocumentRoot "/usr/share/susemanager/www/htdocs"
 
 <Directory "/usr/share/susemanager/www/htdocs">
-    Options Indexes FollowSymLinks
+    Options FollowSymLinks
```

Comment:
If I remember correctly we allow listing basically because of the GPG key and other customer custom file that may be added to the server's public folder. But maybe MC knows more.

Reply:
if it's the case:
- customer custom file should are in `/srv/www/htdocs/pub` (managed by https://github.com/uyuni-project/uyuni/blob/29f2e88abb6a18879b5387696efc62e18a67a1c7/spacewalk/config/etc/httpd/conf.d/z-public.conf )
- gpg keys by /etc/apache2/conf.d/uyuni-build-keys.conf

so changes should be fine